app/src/main/java/de/westnordost/streetcomplete/quests/AbstractCreateNoteFragment.kt

```diff
@@ -37,10 +37,16 @@ abstract class AbstractCreateNoteFragment : AbstractBottomSheetFragment() {
         }
 
         noteInput.addTextChangedListener(TextChangedWatcher { updateOkButtonEnablement() })
+        okButton.isEnabled = false
         okButton.setOnClickListener { onClickOk() }
 
         updateOkButtonEnablement()
     }
+    
+    override fun onViewStateRestored(savedInstanceState: Bundle?) {
```

Comment:
I'm not totally sure that this is the right callback to hook into (whether layout is finished by this point); if it doesn't work we can switch to onStart or onResume instead, or I'll hunt more to see if there's a better callback.